apps/mobile/src/features/Onboarding/Onboarding.container.test.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
-import { Onboarding } from './Onboarding.container'
 import { fireEvent, render } from '@/src/tests/test-utils'
+import { Onboarding } from './Onboarding.container'
```

Comment:
we are no longer loading test-utils in setupFilesAfterEnv and in some tests this was causing issues.